src/models/db/conditionFulfillment.js

```diff
@@ -0,0 +1,31 @@
+/* stub class added for body parsing middleware in app.js */
+'use strict'
+
+const Model = require('five-bells-shared').Model
+const PersistentModelMixin = require('five-bells-shared').PersistentModelMixin
+const JsonField = require('sequelize-json')
+const validator = require('../../services/validator')
+const Sequelize = require('sequelize')
+const sequelize = require('../../services/db')
+const Transfer = require('./transfer').Transfer
+
+class ConditionFulfillment extends Model {
+  /*
+   * This will be expanded when we fully decouple fulfillments from transfers
+   */
+}
+
+PersistentModelMixin(ConditionFulfillment, sequelize, {
+  id: { type: Sequelize.INTEGER, primaryKey: true, autoIncrement: true },
+  transfer_id: Sequelize.UUID,
+  type: Sequelize.ENUM('execute', 'cancel'),
```

Comment:
So this may be a little counterintuitive, but this commit is only meant to add body parsing middleware to the `/fulfillments` endpoint (so that we validate the JSON being passed). I added this db model to be consistent with the other endpoints that use this middleware, and because we will probably use it when we fully separate `fulfillment` from `transfer`. If the user passes a fulfillment as part of a transfer, nothing happens (the schema still allows it, but the business logic doesn't use it), but if they put a fulfillment to the `/fulfillments` endpoint, we persist it in the `transfers` table. In other words, the frontend logic is _mostly_ decoupled, but the backend logic is not yet decoupled. I realize this is not ideal, but I can't progress further until we decide:
1. Do we want to fully decouple these components?
2. Do we need to support backwards compatibility?